package.json

```diff
@@ -74,11 +74,11 @@
     "@apollo/client": "^3.1.5",
     "@babel/runtime": "^7.27.0",
     "@coinbase/wallet-sdk": "^3.3.0",
-    "@cowprotocol/app-data": "^3.2.0",
+    "@cowprotocol/app-data": "^3.3.0",
```

Comment:
There was also a app-data bump on latest cow-sdk. Need to update it here too to keep both in sync.